Makefile

```diff
@@ -28,7 +28,7 @@ ifndef DOCKER_HUB_REGISTRY_IMG
     $(warning DOCKER_HUB_REGISTRY_IMG not defined, using '$(DOCKER_HUB_REGISTRY_IMG)' instead)
 endif
 ifndef BASE_REGISTRY_IMG
-    BASE_REGISTRY_IMG := docker.io/portworx/px-operator-registry:23.5.0
+    BASE_REGISTRY_IMG := docker.io/portworx/px-operator-registry:23.6.0
```

Comment:
The base image should be 23.5.0. The base registry is the latest released version. It acts as the base for building the test catalog with the unreleased 23.6.0 bundle.

Reply:
Oh, I see -- I thought it only influenced the container's name when one did `make container`

I'll roll it back...